minikube/minikube.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash 
+set -x
+set -eu
+set -o pipefail
+
+up_registry () {
+  docker run -d -p 5000:5000 --restart=always --name registry registry:2
+}
+
+deploy_es () {
+  mkdir -p kube_es
+  pushd kube_es
+  for YAML in service-account.yaml es-svc.yaml es-rc.yaml ; do
```

Comment:
It would be greate to place this configuration files statically into `minikube/elasticsearch` to be sure that everything works.